doc/source/actions.rst

```diff
@@ -357,6 +357,13 @@ The ``merge`` action merges the pull request into its base branch. The
        Note that the ``rebase`` method has many drawbacks, see :ref:`strict
        rebase`.
 
+   * - ``priority``
+     - positive integer or one of ``low``, ``default``, ``high``
```

Comment:
```suggestion
     - positive integer or``low`` or ``medium`` or ``high``
```

`default` does not sound like it's between `high` and `low` and I'm sure that what you meant!